OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -1014,6 +1014,68 @@ internal static class DreamProcNativeRoot {
             return new DreamValue(obj.ObjectDefinition.HasProc(procName) ? 1 : 0);
         }
 
+        [DreamProc("hearers")]
+        [DreamProcParameter("Depth", Type = DreamValueTypeFlag.Float)]
+        [DreamProcParameter("Center", Type = DreamValueTypeFlag.DreamObject)]
+        public static DreamValue NativeProc_hearers(NativeProc.Bundle bundle, DreamObject? src, DreamObject? usr) { //TODO: Change depending on center
```

Comment:
what's this todo

Reply:
ah that's copied over from `viewers()`. I left it in because whatever implementation that proc does should do a similar thing to hearers I think

Reply:
afaik the center arg doesn't work correctly and I don't know how to fix it